src/software/thunderscope/replay/replay_file_debugging_script.py

```diff
@@ -88,9 +91,18 @@ def __sort_replay_chunks(file_path: str):
     for replay_file_name in sorted_chunks:
         try:
             total_line_num += read_one_chunk(replay_file_name)
-        except Exception as e:
+
+        except gzip.BadGzipFile as e:
+            print(
+                "replay file is a invalid gzip file:{} with exception: {}".format(
+                    replay_file_name, e
+                )
+            )
+        except Exception:
             print(f"Cannot read {replay_file_name}")
-            print("Please note that the following traceback is ignored. The program is still running")
+            print(
+                "Please note that the following traceback is ignored. The program is still running"
+            )
             print(traceback.format_exc())
```

Comment:
nit: would be helpful to include the exception error in one of these print statements as well.

Reply:
it is indeed printed. line 94 should actually print the entire traceback containing the exception name and along with the stack frame.